addons/mail/static/tests/qunit_suite_tests/components/message_tests.js

```diff
@@ -963,7 +968,13 @@ QUnit.test('message should not be considered as "clicked" after clicking on noti
         notification_status: 'exception',
         notification_type: 'email',
     });
-    const { createThreadViewComponent, messaging } = await start();
+    const { createThreadViewComponent, messaging } = await start({
+        services: {
+            // intercept the mail_resend_message_action which is not registered
```

Comment:
Why not actually registering it instead?

Reply:
Because it relies on a model that is not yet fetched by the `getModelDefinitions` method. Therefore I think that importing a registry, hardcoding an action, fetch another model is quiet cumbersome for something that is not even needed.